sklearn/preprocessing/_data.py

```diff
@@ -291,7 +292,8 @@ class MinMaxScaler(TransformerMixin, BaseEstimator):
     <sphx_glr_auto_examples_preprocessing_plot_all_scaling.py>`.
     """
 
-    def __init__(self, feature_range=(0, 1), copy=True):
+    @_deprecate_positional_args
+    def __init__(self, *, feature_range=(0, 1), copy=True):
```

Comment:
I'm happy with `feature_range` positional or not.

Reply:
Positional isn't ambiguous I would say so +1 for that